cmd/build.go

```diff
@@ -0,0 +1,203 @@
+package cmd
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/CircleCI-Public/circleci-cli/settings"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var dockerAPIVersion = "1.23"
+
+var job = "build"
+var nodeTotal = 1
+var checkoutKey = "~/.ssh/id_rsa"
+var skipCheckout = true
+
+func newBuildCommand() *cobra.Command {
+
+	command := &cobra.Command{
+		Use:   "build",
```

Comment:
Since this command is still running a job (not a full build) I've been wondering if we should rename it to either `run` or `job run`.  We can always add `build` as an alias for backwards compatbility.